src/routes/safe/components/Apps/components/AppCard/index.tsx

```diff
@@ -42,8 +42,13 @@ const AppName = styled(Title)`
 `
 
 const AppDescription = styled(Text)`
-  height: 71px;
   text-align: center;
+  height: 72px;
+  line-height: 24px;
+  overflow: hidden;
+  display: -webkit-box;
+  -webkit-line-clamp: 3;
+  -webkit-box-orient: vertical;
```

Comment:
This won't work in Firefox, will it?

Reply:
Yes! Actually it's working in FF.
![image](https://user-images.githubusercontent.com/622217/123993487-d831be00-d9a2-11eb-992d-2239b284cc40.png)

I've tested it in Brave, Opera, Chrome and FF. 
https://drafts.csswg.org/css-overflow-3/#webkit-line-clamp

Reply:
Oh, awesome! Let's merge it then. Thanks, Agustin!